adminSiteClient/GdocsPreviewPage.tsx

```diff
@@ -52,9 +48,19 @@ export const GdocsPreviewPage = ({ match, history }: GdocsMatchProps) => {
     }>({ original: undefined, current: undefined })
     const originalGdoc = gdoc.original
     const currentGdoc = gdoc.current
-    const setCurrentGdoc = (current: OwidGdocInterface | undefined) => {
+    const putCurrentGdoc = useDebounceCallback((current: OwidGdocInterface) => {
         cancelAllRequests()
-        setGdoc({ original: gdoc.original, current })
+        store.update(current)
+    }, 2000)
+    const setCurrentGdoc = (
+        current: OwidGdocInterface | undefined,
+        skipUpdatingDb = false
+    ) => {
+        setGdoc({ original: originalGdoc, current })
```

Comment:
I think this messes up with the "Saving..." status. Is `originalGdoc` ever updated post page load?